### ISSUE TITLE
Mobile version broken

### ISSUE THREAD

**Issue by guizmoau** [roundcube/roundcubemail#7404]
Hello,

I am running roundcube 1.4.4 with elastic skin.
When I connect on a mobile phone there is an iframe loading with a white page over the screen.

I can't go bakc to the left with arrow, it forces me back to that white screen, on the right I have a acces to a menu.

That iframe is 
id="aria-label-mailpreviewframe"
<iframe name="messagecontframe" id="messagecontframe" aria-labelledby="aria-label-mailpreviewframe" src="./?_task=mail&amp;_action=plugin.watermark" title="Prévisualisation des courriels"></iframe>

I am wondering how can I make this to work with mobile, I thought that roundcube / elastic was responsive already.

**Comment by alecpl** [comment]
What browser/device/OS version? Could you provide a screenshot/screencast? Steps to reproduce.

**Comment by guizmoau** [comment]
browser chrome with mobile via device toolbar (when i rotate the device I can see the left column menu so it is okay)
browser brave / chrome on android phone
server is ubuntu 18.04 roundcube 1.4.4

I have attached the white screen I get stuck with when the page finish loading...then when I try to click the top left arrow it forces me back on that white screen, so I can't access the left side menu. Instead I have a right side menu.

![Screenshot 2020-05-26 at 09 46 14](https://user-images.githubusercontent.com/5398689/82874278-fc1c0800-9f35-11ea-8554-bce5d3e1e2c6.png)

![Screenshot 2020-05-26 at 09 49 25](https://user-images.githubusercontent.com/5398689/82874408-338ab480-9f36-11ea-8506-29ba8ec4e96c.png)

**Comment by guizmoau** [comment]
Sometimes the bottom right next button is grey so I can't access any emails at all....

![Screenshot 2020-05-26 at 09 51 11](https://user-images.githubusercontent.com/5398689/82874766-b9a6fb00-9f36-11ea-9e9a-8ae2a662792a.png)

Looking at the code it is clearly that watermark iframe that is loading and covering the screen, I guess it supposed to show the list of emails in a mobile friendly-way applying over the normal interface.

<iframe name="messagecontframe" id="messagecontframe" aria-labelledby="aria-label-mailpreviewframe" src="./?_task=mail&amp;_action=plugin.watermark" title="Prévisualisation des courriels"></iframe>

**Comment by alecpl** [comment]
What Android and Chrome version? Any chance to see if there's any javascript error in browser console?

**Comment by guizmoau** [comment]
Chrome Version 83.0.4103.50 (Official Build) beta (64-bit)

![Screenshot 2020-05-26 at 10 50 09](https://user-images.githubusercontent.com/5398689/82880797-3047f680-9f3f-11ea-834d-d99f1d162b9d.png)

I have to verify for android version

**Comment by alecpl** [comment]
I see no issues when using mobile mode of Chrome 81 on a desktop computer. I don't have an Android phone at the moment, and I don't have Chrome 83.

**Comment by guizmoau** [comment]
I see.

Do you know about that iframe loading with watermark.html ?
Would I had a way to disactivate it or anyway ?
How can I continue to debug on my own, the error console on the web browser doesnt help  ?

Thanks a lot for your help @alecpl

**Comment by guizmoau** [comment]
this is my list of enabled plugins from config/config.inc.php file

I have tryed to re-organize a bit and disable a few without any luck yet

$config['plugins'] = array(
        'elastic4mobile',
        'html5_notifier',
        'mobile',
        'archive',
        'zipdownload',
        'password',
        'jqueryui',
        'managesieve',
        'persistent_login',
        'swipe',
        'carddav',
        'account_details',
        'calendar',
        'enigma',
        'contextmenu',
        'twofactor_webauthn',
        'customizr',
);

**Comment by alecpl** [comment]
I'd propose to try with disabled all plugins. The iframe is supposed to display the email message content, so you can't get rid of it.

**Comment by guizmoau** [comment]
would it be a standard test "demo" version of roundcube online somewhere that I could try to see if it come from my specific versions and setup ?

**Comment by alecpl** [comment]
There's no demo. https://kolabnow.com/ is using recent Roundcube version with slightly modified Elastic and is free for 30 days. Maybe there are other free services, I don't know. Did you try using a different mobile device?

**Comment by guizmoau** [comment]
ok, I found it !

The plugin customizr was the one in fault.
Then I found out that those two options :
// define a custom watermark image (relative or absolute URL)
$config['custom_watermark_image'] = '';

// define a custom URI to be displayed instead of the empty watermark page
$config['custom_watermark_uri'] = '';

I had defined the first one (custom_watermark_image) but the second one I left empty.

I guess this is to show a logo instead of whitepage when there is no mail...but not sure why it needs two parameters.

Anyway it is solved that way ! :)

Thanks again @alecpl for your support and guidance
